draft-ietf-ppm-dap.md

```diff
@@ -1774,6 +1789,20 @@ error `unrecognizedTask`.
 Next, it checks if it recognizes the indicated aggregation job ID. If not, it
 MUST abort with error `unrecognizedAggregationJob`.
 
+Next, the Helper checks if the continuation step indicated by the request is
+correct. (For the first `AggregationJobContinueReq` the value should be `1`;
+for the second the value should be `2`; and so on.) If the Leader is one step
+behind (e.g., the Leader has resent the previous HTTP request), then the Helper
+MAY attempt to recover by sending HTTP 202 status Accepted. In this case it
+SHOULD verify that the contents of the `AggregationJobContinueReq` are
+identical to the previous message (see {{aggregation-step-skew-recovery}}).
+Otherwise, if the step is incorrect, the Helper MUST abort with error
+`stepMismatch`.
+
+Next, the Helper checks whether the aggregation job is already in progress from
+a prior continuation or initialization request. If so, it MUST abort with error
+`stepMismatch`.
```

Comment:
Can you sketch out the scenario this is meant to catch? Per the previous paragraph, it should be OK for the leader to send the same `AggregationJobContinueReq` twice for idempotency reasons.

Reply:
Ah, you're right. I was thinking this would handle the case where the leader sends two different POST requests, but as you stated the idempotency check would catch this.

I will strike this step, and revert the reordering of steps (as it's not necessary).